packages/ui/src/message-room-details-dialog/components/EditDescriptionModal.jsx

```diff
@@ -11,7 +12,45 @@ import {
   ModalTopic
 } from "../MessageRoomDetailsDialog.styled";
 
-const EditDescriptionModal = ({ closeEdit }) => {
+const EditDescriptionModal = ({ closeEdit, roomData }) => {
+  const roomDescription = roomData.data.description;
+  const descriptionRef = useRef();
+
+  // to update data
+  const handleDesc = () => {
+    const org_id = roomData.data.org_id;
+    const member_id = roomData.data.created_by;
+    const room_id = roomData.data._id;
+
+    const editedDescription = {
+      room_name: roomData.data.room_name,
+      description: descriptionRef.current.value,
+      topic: roomData.data.topic,
+      is_private: roomData.data.is_private,
+      is_archived: roomData.data.is_archived
+    };
+
+    fetch(
+      `https://chat.zuri.chat/api/v1/org/${org_id}/members/${member_id}/rooms/${room_id}`,
+      {
+        method: "PUT",
+        headers: {
+          "Content-Type": "application/json",
+          Accept: "application/json"
+        },
+        body: JSON.stringify(editedDescription)
+      }
+    )
+      .then(res => {
+        if (!res.ok) {
+          throw Error("Could not change description");
+        }
+      })
+      .catch(err => {
```

Comment:
don't add a catch if you're not going to use it

Reply:
I am actually still making changes, I thought I made this a draft. I'll make all these changes and make a proper PR soon